### PR TITLE
Fix: Redesign auth view(#815)

### PR DESCRIPTION
**Pull request by Ram81**
@aka-jain please review!
I've done some changes to login & signup page but there is still work to do in design of both of them,
and also I wanted to know do you want the footer to be there in these pages?

### CODE REVIEW

**Comment by aka-jain** [comment]
@Ram81 please use classes that are already present in landing and about us page. Don't change styles of auth CSS.

**Comment by Ram81** [comment]
@aka-jain  I needed to change the auth-container class otherwise the alignment was not as we expected do you want me to use grad-container class instead of auth-container like we did in about-us page?

**Comment by aka-jain** [comment]
@Ram81 yes please, use those classes.

**Comment by Ram81** [comment]
@aka-jain but if I use grad-conatiner class then alignment of the login & signup view to be loaded & also the spacemen container gets disturbed please suggest what should I do?

**Comment by aka-jain** [comment]
That's why I've created a new issue for auth design. So you have to redesign the auth view again and use classes as we used in about us page. Do that first please.

**Comment by Ram81** [comment]
@aka-jain I've used existing css classes only now & what about the footer?

**Comment by aka-jain** [comment]
@Ram81 Add that too.

File: frontend/src/views/web/auth.html [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
-            <div class="reg-alert col s12 m4 offset-m4 align-left text-highlight w-300">{{auth.regMsg}}</div>
-            <!-- render login/signup form here -->
-            <div class="ui-view"></div>
-        </div>
-    </div>
-</section>
+<div class="white-bg view-container flex-body">
+	<main-header></main-header>
+	<!-- EvalAI into -->
+	<div class="ev-container grad-container content">
```
Comment (aka-jain) [inline frontend/src/views/web/auth.html]: @Ram81 make a seperate div for `content` class.
p.s. just for readeability.

Comment (Ram81) [inline frontend/src/views/web/auth.html]: @aka-jain sure

**Comment by Ram81** [comment]
@aka-jain please review!

**Comment by deshraj** [comment]
@Ram81 can you please share screenshot?

**Comment by Ram81** [comment]
log

**Comment by aka-jain** [comment]
@Ram81 That's not looking good. Let me redesign this page. Will share the auth design with you soon. 
Thanks for the efforts. I'm closing this PR.

**Pull request closed without merging**